src/videouplinkbandwidthpolicy/NScaleVideoUplinkBandwidthPolicy.ts

```diff
@@ -9,17 +10,25 @@ import VideoUplinkBandwidthPolicy from './VideoUplinkBandwidthPolicy';
 /** NScaleVideoUplinkBandwidthPolicy implements capture and encode
  *  parameters that are nearly equivalent to those chosen by the
  *  traditional native clients, except for a modification to
- *  maxBandwidthKbps described below. */
+ *  maxBandwidthKbps and scaleResolutionDownBy described below. */
 export default class NScaleVideoUplinkBandwidthPolicy implements VideoUplinkBandwidthPolicy {
+  static readonly rid = 'video';
```

Comment:
This needs to be in a docstring somewhere, right? If I write my own policy, I need this to be in the encoding parameters…

Reply:
So this is not relevant unless it is simulcast, the default rid is undefined or "" (in Safari). Maybe `rid` is a bad name here but this is just a dummy parameter since setEncodingParameters requires a string key.

Reply:
That sounds like a great topic to cover in the developer doc for this public interface!